### ISSUE TITLE
Troubleshoot why some records do not update even when import is successful

### ISSUE THREAD

**Issue by karenmajewicz** [geobtaa/geomg#274]
### Today's scenario:

1. I imported a CSV of **282** items. Each had some type of change:
    - all items: new keyword of "bboxSprintJune2023"
    - some items: new Bounding Box
    - some items: new Spatial Coverage
2. The CSV imported and no errors were reported
3. When I searched for "bboxSprintJune2023" in GEOMG or the Geoportal, only **267** items returned. There should have been 282.
4. I checked a few of the items that did not return and could see that they had not been updated by the latest CSV import and did not have the new keyword. I have not determined why these items did not work - they were from various Providers, had a mix of ID formats
5. On a hunch, I deleted the Geometry field and reuploaded. After this, **276** items updated.

Perhaps the remaining items have something wrong with the Bounding Box field. I am attaching the list and will manually try to update them.
[didNotUpdate.csv](https://github.com/geobtaa/geomg/files/11851541/didNotUpdate.csv)

----
###  Previous indications

I think this might be happening regularly, but I am not catchingit. I have had suspicions, though due to this process I do weekly:

1. Week 1: Upload all records harvested from ArcGIS Hubs (~10,000) with that day's date in the Date Accessioned field (ex. `2023-07-01)
2. Week 2: Upload all records harvested from ArcGIS Hubs (~10,010) with that day's date in the Date Accessioned field (ex. `20223-07-07`)
3. Any records that still have `2023-07-01`, I assume were not present in the latest harvest and must be outdated. I unpublish them.

However, every week, there seems to be more records to unpublish than are new. I think that during every upload, a handful of records do not get updated with the new Date Accessioned value even though they do not report as failed during the upload process.

~~Unlike issue #119 , these records do not update in GEOMG nor Solr.~~ This was wrong; see below

**Comment by karenmajewicz** [comment]
Imports that caused this problem:

267 of 282: https://geomg.lib.umn.edu/imports/1029 - Paging through "Success" tabs triggers a Rails error
276 of 282: https://geomg.lib.umn.edu/imports/1031 - Paging through "Success" tabs triggers a Rails error

**Comment by karenmajewicz** [comment]
This appears to be another instance of issue #119. I forgot that doing searches in GEOMG is actually returning Solr results. For example, this record https://geomg.lib.umn.edu/documents/p16022coll230:656/edit actually has "bboxSprintJune2023" in the keywords as shown in the form. However, this never got pushed to Solr, so when I search for it in GEOMG, it does not appear on the list.  [This could be something to document for users.]

**Comment by karenmajewicz** [comment]
I was able to manually update the remaining items by deleting the Bounding Box and Geometry and inserting a new value, mostly `-180,-90,180,90`

I believe that the problem was that these records had easting and westing values of -180 and 180. However, the north and south values were something different that -90 or 90. This resulted in the same error we previously had in in issue #228 . We solved that by converting anything with `-180,-90,180,90` into ENVELOPE(-180,180,90,-90) in the Geometry field instead of a Polygon. However, we missed that something like a Bounding Box with `-180,-80,180,80` will generate this value in Geometry: `POLYGON((-180 80, 180 80, 180 -80, -180 -80, -180 80))`, which is invalid.
